packages/fast-usdc/src/exos/advancer.js

```diff
@@ -217,7 +217,7 @@ export const prepareAdvancerKit = (
             });
           } catch (error) {
             log('Advancer error:', error);
-            statusManager.observe(evidence);
+            statusManager.skipAdvance(evidence, ['PRECONDITIONS_FAILED']);
```

Comment:
This looks good. As discussed offline let's try this approach, [like smart wallet](https://github.com/Agoric/agoric-sdk/blob/8ff98c413c15e9f5945a3734730cdc1cc0db9793/packages/smart-wallet/src/smartWallet.js#L1041):
```suggestion
            statusManager.skipAdvance(evidence, [error.message]);
```

Reply:
Done